tests/beaker_tests/cisco_ip_multicast/test_ip_multicast.rb

```diff
@@ -0,0 +1,90 @@
+###############################################################################
+# Copyright (c) 2017-2018 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+#
+# See README-develop-beaker-scripts.md (Section: Test Script Variable Reference)
+# for information regarding:
+#  - test script general prequisites
+#  - command return codes
+#  - A description of the 'tests' hash and its usage
+#
+###############################################################################
+require File.expand_path('../../lib/utilitylib.rb', __FILE__)
+
+# Test hash top-level keys
+tests = {
+  agent:         agent,
+  master:        master,
+  platform:      'n9k',
+  resource_name: 'cisco_ip_multicast',
+}
+
+# Skip -ALL- tests if a top-level platform/os key exludes this platform
+skip_unless_supported(tests)
+
+# Skip -ALL- tests if feature is not supported by image version
+skip_nexus_image(/I[2-6]/, tests)
+
+# Test hash test cases
+tests[:default] = {
+  desc:           '1.1 Default Properties',
+  title_pattern:  'default',
+  manifest_props: {
+    overlay_distributed_dr: 'default',
+    overlay_spt_only:       'default',
+  },
+  resource:       {}, # no output expected when all defaults
```

Comment:
If the manifest uses `ensure => present` then I would expect the resource to display both properties set to the default value of `false`.

Reply:
done